### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [anicksaha/anicksaha.github.io#2]
The user @carlsonsantana validate your site template "https://anicksaha.github.io/" and found these accessibility errors:
* Page https://anicksaha.github.io/:
  * Issue *753fbd3a-6779-470f-92ad-1f377265c301*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.06:1. Recommendation: change text colour to #6c6c6c.
    * **Selector**: `html > body > div > footer > a`
    * **Context**: `<a href="http://jekyllrb.com/">Jekyll</a>`
  * Issue *c1e2b99f-d806-4c61-9864-dc807217aa91*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.06:1. Recommendation: change text colour to #6c6c6c.
    * **Selector**: `html > body > div > footer`
    * **Context**: `<footer class="site-footer">
    <br>
    Using <a href="h...</footer>`
  * Issue *ca6d3a23-a896-49cf-9a44-3a8d04f83802*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    * **Message**: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
    * **Selector**: `html > body > div > div > div > a`
    * **Context**: `<a href="images/Anick_Amsterdam.jpg">
    
    <img src="images/Anic...</a>`
  * Issue *6a9c1636-2ed3-498a-af20-1f585e5c61d3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.06:1. Recommendation: change text colour to #6c6c6c.
    * **Selector**: `html > body > div > header > nav > div > ul > li:nth-child(4) > a`
    * **Context**: `<a href="" class="page-link">More</a>`
  * Issue *dd6a8b2c-c040-47dc-b348-a4974233cd15*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.06:1. Recommendation: change text colour to #6c6c6c.
    * **Selector**: `html > body > div > header > nav > div > ul > li:nth-child(3) > a`
    * **Context**: `<a href="/blog/" class="page-link">Blog</a>`
  * Issue *25f8524f-367f-42c7-b48b-c9fba11203de*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.06:1. Recommendation: change text colour to #6c6c6c.
    * **Selector**: `html > body > div > header > nav > div > ul > li:nth-child(2) > a`
    * **Context**: `<a href="/misc/" class="page-link">Misc</a>`
  * Issue *319dec4f-2e3d-4cb8-8de3-2af185daded7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.06:1. Recommendation: change text colour to #6c6c6c.
    * **Selector**: `html > body > div > header > nav > div > ul > li:nth-child(1) > a`
    * **Context**: `<a href="/research/" class="page-link">Research</a>`
  * Issue *1e970a3d-b408-4126-bf8c-1233ba6d5130*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > div > header > nav > a`
    * **Context**: `<a href="#" class="menu-icon menu.open">
            <svg viewBox="0 0 ...</a>`
  * Issue *09c8fe9a-6ba4-4826-8e0b-394a4c629138*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.06:1. Recommendation: change text colour to #6c6c6c.
    * **Selector**: `html > body > div > header > a`
    * **Context**: `<a class="site-title" href="/">Anick Saha</a>`
  * Issue *2512ac82-80f9-4aa8-b30f-87d6c1ec8f2e*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
    <meta charset="utf-8...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/bd30a95f-3c54-4be0-8390-233d27a56abf/).
